pyproject.toml

```
@@ -167,7 +175,8 @@ linting = [
 test = [
   "covdefaults",
   "pytest",
-  "pytest-asyncio",
+  "pytest-asyncio; python_version < \"3.13\"",
+  "pytest-asyncio @ git+https://github.com/provinzkraut/pytest-asyncio.git@shutdown-asyncgens; python_version >= \"3.13\"",
```

```suggestion
  "pytest-asyncio",
```
Revert this once the PR has been merged.